app/components/pages/createPoll/index.js

```diff
@@ -0,0 +1,234 @@
+                                  variant="outlined"
+                                />
+                              </Box>
+
+                              <Box
+                                position="absolute"
+                                top="0"
+                                right="0"
+                                pr={2}
+                                pt={2}
+                              >
+                                <Button
+                                  type="button"
+                                  onClick={() => arrayHelpers.remove(index)}
+                                >
+                                  X
+                                </Button>
+                              </Box>
+                            </Box>
+                          ))}
+                        </Box>
+
+                        <Box
+                          width={1}
+                          mt={15}
+                          mb={15}
+                          display="flex"
+                          flexDirection={['column', 'row']}
+                          justifyContent="center"
+                        >
+                          <Box
+                            ml={2}
+                            display="flex"
+                            justifyContent={['center', 'left']}
+                          >
+                            <Button
+                              startIcon={<AddIcon fontSize="small" />}
+                              variant="contained"
+                              disabled={
+                                values.options.length >= upperLimitForOptions
+                              }
+                              color="secondary"
+                              type="button"
+                              onClick={() => arrayHelpers.push('')}
+                            >
+                              {optionButtonName}
+                            </Button>
+                          </Box>
+                          <Box
+                            ml={2}
+                            my={[2, 0]}
+                            display="flex"
+                            justifyContent={['center', 'left']}
+                          >
+                            <Button
+                              variant="contained"
+                              color="secondary"
+                              type="submit"
+                              startIcon={<SaveIcon />}
+                            >
+                              {buttonName}
+                            </Button>
+                          </Box>
+                          <Box
+                            ml={2}
+                            my={[2, 0]}
+                            display="flex"
+                            justifyContent={['center', 'center', 'left']}
+                          >
+                            <Button startIcon={<ClearIcon fontSize="small" />}>
```

Comment:
remove fontSize="small"
i know it's used in announcement

Reply:
yes from annoucement i got may be it required 
now resolved